src/mixins/nr-affiliation-mixin.ts

```diff
@@ -103,7 +120,7 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
         await AuthServices.removeNrAffiliation(accountId, nr.nrNum).catch(() => null)
       }
 
-      throw new Error('Unable to create new business')
```

Comment:
`Error()` can be called with or without `new`, so I went with the cleaner approach.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error